tools/cldr-apps/js/src/esm/cldrAddAlt.mjs

```diff
@@ -21,11 +20,11 @@ function addButton(containerEl, xpstrid) {
     addAltWrapper.setXpathStringId(xpstrid);
   } catch (e) {
     console.error("Error loading Add Alt vue " + e.message + " / " + e.name);
-    notification.open({
-      message: `${e.name} while loading AddAlt.vue`,
-      description: `${e.message}`,
-      duration: 0,
-    });
+    cldrNotify.open(
+      `${e.name} while loading AddAlt.vue`,
```

Comment:
```suggestion
      `Internal Error: ${e.name} while loading AddAlt.vue`,
```

AddAlt.vue isn't going to make any sense to users, so I think it's best to preface it with "Internal Error" which is the most important part for them to know.